test/ejabberd_tests/rebar.config

```diff
@@ -4,7 +4,7 @@
 {require_otp_vsn, "1[789]"}.
 
 {deps, [
-        {escalus, ".*", {git, "git://github.com/esl/escalus.git", "cf285c0"}},
+        {escalus, ".*", {git, "git://github.com/esl/escalus.git", {branch, "add-caps-to-stream-features"}}},
```

Comment:
That's what we usually do :) the branch ref is only while developing and waiting for changes in escalus to be merged. As soon as the esl/escalus#120 is merged I'll update rebar.config with relevant commit's hash.